### ISSUE TITLE
Don't require a virtual environment

### ISSUE THREAD

**Issue by YakDriver** [plus3it/gravitybee#74]
**Is your feature request related to a problem? Please describe.**
Yes, it can be a pain, depending on the situation (cough, cough Azure Pipelines), to get a Python virtual environment working properly. It would be better if gravitybee could set up its own virtual environment as part of the build process rather than forcing the user to do that beforehand.

**Describe the solution you'd like**
Run gravitybee outside of a virtual environment and it generates a working standalone.

**Describe alternatives you've considered**
The alternative is having the user create the virtual environment, which is what happens now.

**Additional context**

**Comment by solubrew** [comment]
I'm having trouble understanding the virtual environment configuration for gravitybee.  i'm trying to build from an Anaconda virtual environment but I keep getting the following error message
gravitybee - ERROR - No virtual environment directory detected!

**Comment by YakDriver** [comment]
@solubrew There may be deeper incompatibilities between Anaconda virtual environments and virtualenv but the error that you're getting is caused by the "VIRTUAL_ENV" environment variable not being set. (`virutalenv` sets it when you activate a virtual environment.)

However, activating a non-Anaconda virtual environment is pretty simple (see Fix #2 in [this article](https://medium.com/faun/azure-pipelines-and-python-virtual-environments-fdc1089656db)) and getting GravityBee to work with your virtual environment might also be simple.

```console
$ export VIRTUAL_ENV="$HOME/anaconda/envs/<name of your env>"
$ export PATH="$VIRTUAL_ENV/bin:$PATH"
$ unset PYTHON_HOME
```

Then try running GravityBee. I'll open an issue for this and please let me know if it works or not.
